conf/stressor/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+export PATH=/usr/bin:$PATH
+PAYLOAD_GEN=${PAYLOAD_GEN:-fixed}
+DISTRIBUTION=${DISTRIBUTION:-fixed}
+PAYLOAD_SIZE=${PAYLOAD_SIZE:-100}
+MSG_PER_SEC=${MSG_PER_SEC:-1}
+OUTPUT_FORMAT=${OUTPUT_FORMAT:-stdout}
+
+OUT_FILE=${OUTFILE:-""}
+if [ -z "$OUTFILE" ]; then
+    OUTFILE="> $OUTFILE"
+fi
+
+log-stressor \
+    --payload-gen   $PAYLOAD_GEN \
+    --distribution  $DISTRIBUTION \
+    --payload_size  $PAYLOAD_SIZE \
+    --msgpersec     $MSG_PER_SEC \
+    --msgpersec     $MSG_PER_SEC \
+    --output-format $OUTPUT_FORMAT \
```

Comment:
you called the flag `format` here https://github.com/ViaQ/cluster-logging-collector-benchmarks/pull/4/files#diff-62dd2450e04e29eea4f8e153c27cf9747b325911ed3ef7f03276891c01757a70R35 ... so either fix that to be `format` or change the flag (in golang) to be output format

Reply:
changing flag to "output-format"